build.gradle

```diff
@@ -36,6 +36,9 @@ plugins {
 repositories {
     mavenCentral()
     mavenLocal()
+    maven {
+        url "https://yahoo.bintray.com/maven"
```

Comment:
LGTM there is one place that is a little confused. Is this update the source of any package that depends on Yahoo?

Reply:
@tuteng 
This is needed to use [pulsar-client-auth-athenz(OSS Athenz)](https://github.com/apache/pulsar-manager/pull/286/files#diff-c197962302397baf3a4cc36463dce5eaR114).
If there is no definition, Jenkins build will fail.

The same definition is in pom.xml for [Pulsar](https://github.com/apache/pulsar/blob/5d02c70b674a36e5bd36c60e488737db5740d21e/pom.xml#L1723) and [Athenz](https://github.com/yahoo/athenz/blob/57255818c384acce16cf36944afff10bc26e68f1/pom.xml#L286).